openfl-tutorials/experimental/Workflow_Interface_401_MNIST_Aggregator_Validation_Ray_Watermarking.ipynb

```diff
@@ -0,0 +1,920 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "dc13070c",
+   "metadata": {},
+   "source": [
+    "# Workflow Interface 401: Aggregator validation with a watermark dataset using Ray\n",
+    "\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/intel/openfl/blob/develop/openfl-tutorials/experimental/Workflow_Interface_401_MNIST_Aggregator_Validation_Ray_Watermarking.ipynb)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "8f28c451",
+   "metadata": {},
+   "source": [
+    "Authors:\n",
```

Comment:
The author list should be removed considering this is a consolidation of prior work. All coauthors will be credited when OpenFL 1.6 is released (see our [OpenFL 1.5 release announcement](https://github.com/securefederatedai/openfl/releases/tag/v1.5) for reference)

Reply:
We removed the author list with the last commit